frontend/packages/shipwright-plugin/console-extensions.json

```diff
@@ -103,6 +191,21 @@
       },
       "component": { "$codeRef": "pages.BuildListPage" }
     },
+    "flags": {
+      "required": ["SHIPWRIGHT_BUILD_V1ALPHA1"],
+      "disallowed": ["SHIPWRIGHT_BUILDRUN"]
+    }
```

Comment:
I believe we don't need these feature flags because the list component is only used when a user opens a specific URL. And if the user does it, it will still work to show the Alphav1 data. This might be useful if someone bookmarked or pinned this resource.

```suggestion
```